spec/views/admin/websites/show.html.erb_spec.rb

```diff
@@ -1,28 +1,23 @@
-require 'rails_helper'
+require 'spec_helper'
 
-RSpec.describe "admin/websites/show", type: :view do
-  before(:each) do
-    @admin_website = assign(:admin_website, Website.create!(
-      :domain => "Domain",
-      :title => "Title",
-      :logo => "Logo",
-      :stylesheet => "Stylesheet",
-      :header => "MyText",
-      :footer => "MyText",
-      :redirect_url => "Redirect Url",
-      :state => "State"
-    ))
-  end
+describe "admin/websites/show", type: :feature do
+  let!(:account) { create :account }
+  let!(:website) { create :website }
 
   it "renders attributes in <p>" do
-    render
-    expect(rendered).to match(/Domain/)
-    expect(rendered).to match(/Title/)
-    expect(rendered).to match(/Logo/)
-    expect(rendered).to match(/Stylesheet/)
-    expect(rendered).to match(/MyText/)
-    expect(rendered).to match(/MyText/)
-    expect(rendered).to match(/Redirect Url/)
-    expect(rendered).to match(/State/)
+    account.update(role: :admin)
```

Comment:
You made integration testing in views testing
http://guides.rubyonrails.org/testing.html#testing-views

just move it into integration test and for view testing you can fix the previous one,
simplify them you can check only most important fields.